tests/e2e_tests/e2e_stress_test_config.yml

```
@@ -18,3 +18,4 @@ LANGUAGES:
 
 FEATURE_FLAGS:
   USE_LAZY_LOADING: True
+  USE_SERVER_SIDE_CLUSTERING: True
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)